utils/host_utils.cpp

```diff
@@ -146,6 +146,13 @@ void setQudaDefaultMgTestParams()
   }
 }
 
+bool isPCSolution(QudaSolutionType solution_type) {
```

Comment:
This definition could be moved into `utils/host_utils.h` and then get used in `utils/staggered_host_utils.cpp`

Reply:
7052e04